packages/chaire-lib-backend/src/models/db/tokens.db.queries.ts

```diff
@@ -109,11 +126,27 @@ const getUserByToken = async (token: string) => {
     }
 };
 
+async function cleanExpiredApiTokens() {
+    try {
+        const tokenLifespan = new Date();
+        const rowsToDelete = await knex(tableName).where('expiry_date', '<', tokenLifespan);
```

Comment:
Wonder if it would be preferable to use the NOW() function of the database instead of doing the data in TS

Reply:
Both work. I changed it to "NOW()" so today's date wouldn't be implicit